b2b-sample/apps/business-app/components/sections/sections/EasyMeetings/blogSection/otherComponents/newsComponent.tsx

```diff
@@ -17,40 +17,44 @@
  */
 
 import { getCurrentDate } from "@b2bsample/shared/util/util-common";
-import Image from "next/image";
-import React from "react";
-import { Panel } from "rsuite";
-import image1 from "../../../../../../../libs/business-app/ui-assets/src/lib/images/news/news1.jpeg";
-import image2 from "../../../../../../../libs/business-app/ui-assets/src/lib/images/news/news2.jpeg";
-import image3 from "../../../../../../../libs/business-app/ui-assets/src/lib/images/news/news3.jpeg";
-import image4 from "../../../../../../../libs/business-app/ui-assets/src/lib/images/news/news4.jpeg";
+import Image, { StaticImageData } from "next/image";
+import { Stack } from "rsuite";
+import image1 from "../../../../../../../../libs/business-app/ui-assets/src/lib/images/news/news1.jpeg";
+import image2 from "../../../../../../../../libs/business-app/ui-assets/src/lib/images/news/news2.jpeg";
+import image3 from "../../../../../../../../libs/business-app/ui-assets/src/lib/images/news/news3.jpeg";
+import image4 from "../../../../../../../../libs/business-app/ui-assets/src/lib/images/news/news4.jpeg";
+
+interface NewsComponentInterface {
+    header: string,
+    body: string
+}
 
 /**
  * 
  * @param prop - header (header text), body (body text)
  *
  * @returns Single news component
  */
-export default function NewsComponent(prop) {
+export default function NewsComponent(props: NewsComponentInterface) {
 
-    const { header, body } = prop;
+    const { header, body } = props;
 
     return (
-        <div>
-            <Panel>
-                <Image src={ selectImage() } height={ 800 } width={ 1000 } alt="news image"/>
-                <p><br /></p>
+        <Stack spacing={ 20 }>
+            <Image src={ selectImage() } height={ 250 } width={ 300 } alt="news image" />
+            <Stack direction="column" alignItems="flex-start">
                 <p>{ getCurrentDate() }</p>
                 <br />
                 <h4>{ header }</h4>
                 <p>{ body }</p>
-            </Panel>
-        </div>
+            </Stack>
+
```

Comment:
Let's remove the unwanted new line.

Reply:
Fixed with https://github.com/wso2/samples-is/pull/416/commits/c4229e64c82f57a16a579a24259eafab0e6cc56f